### DISCUSSION TITLE
Is it possible to show the AR button on Google Chrome browsers on iOS devices?

### DISCUSSION THREAD

**Discussion by hd-esteban** [google/model-viewer#1911] [Q&A]
If yes, since which version it is possible?

**Comment by elalish** [comment]
I forget which version this was added in, but I'd always recommend using the latest. Try the `quick-look-browsers` attribute: https://modelviewer.dev/examples/augmented-reality.html

**Comment by hd-esteban** [comment]
thank you for your quick response, I'll try it, just one more question, few months ago I read something about the Content-Type, I think USDZ files should be retrieved using a specific value or something like that, do you have any documentation for it?

**Comment by elalish** [comment]
I think you'll have to check apple's documentation for that. We merely pass your `ios-src` URI to quickLook.

**Comment by hd-esteban** [comment]
Hi @elalish, thanks for your suggestion, one more question, which is the Content-Type that we should use for the gtfl files on Android devices?

**Comment by elalish** [comment]
I believe what you're looking for is just below the image here: https://docs.fileformat.com/3d/gltf/
